kategory/src/main/kotlin/kategory/instances/IdInstances.kt

```diff
@@ -41,3 +41,14 @@ interface IdInstances :
             fa.ev().value
 
 }
+
+object IdSemigroupK : SemigroupK<Id.F>, GlobalInstance<SemigroupK<Id.F>>() {
+    override fun <A> combineK(x: HK<Id.F, A>, y: HK<Id.F, A>): HK<Id.F, A> = y
```

Comment:
Typeclass method implementations should always return the most concrete type close to the kind. In this case: `Id<A>` and same for all other methods. This removes the need of down-casting explicitly for users when using the instance directly.